### DISCUSSION TITLE
Wrong "lowest value" on SRT 321 thermostat, HA or Zwave-js-ui problem ?

### DISCUSSION THREAD

**Discussion by minigranis** [zwave-js/zwave-js-ui#3920] [General]
Hello ! 
Newbie here trying to get an annoying thing fixed, when using the physical dial on my SRT 321 Z wave thermostats the lowest value is 5 degrees Celsius. When using Home Assistant the lowest value is 7. I can set it to 5 manually when using the UI in Zwave-js-ui.
I would like the lowes value in HA also to be 5 (as it should), is this a problem within Home Assistant or in Zwave-ju-ui ?

Data on the theremostat can be found here:
https://manuals-backend.z-wave.info/make.php?lang=en&sku=MAX10U-734&cert=ZC10-16015001&type=mini

**Comment by kpine** [comment]
https://github.com/zwave-js/node-zwave-js/discussions/7202
